tests/go/go-kafka-serde/protobuf-serializer.go

```diff
@@ -0,0 +1,60 @@
+package main
+
+import (
+	"github.com/confluentinc/confluent-kafka-go/v2/schemaregistry"
+	"github.com/confluentinc/confluent-kafka-go/v2/schemaregistry/serde"
+	"github.com/confluentinc/confluent-kafka-go/v2/schemaregistry/serde/protobuf"
+	log "github.com/sirupsen/logrus"
+)
+
+type ProtobufSerializer struct {
+	ser   *protobuf.Serializer
+	deser *protobuf.Deserializer
+}
+
+func NewProtobufSerializer(srClient *schemaregistry.Client) (s *ProtobufSerializer, err error) {
+	ser, err := protobuf.NewSerializer(*srClient, serde.ValueSerde, protobuf.NewSerializerConfig())
+
+	if err != nil {
+		return nil, err
+	}
+
+	deser, err := protobuf.NewDeserializer(*srClient, serde.ValueSerde, protobuf.NewDeserializerConfig())
+
+	if err != nil {
+		return nil, err
+	}
+
+	deser.ProtoRegistry.RegisterMessage((&Payload{}).ProtoReflect().Type())
```

Comment:
I'll need to add a readme to instruct the user to set `$GOPATH` when calling `make all`

As for the second part... that seems weird.  "It works fine on my machine"

Reply:
Problem was on my environment. Not sure why that extends to the container, but I'm guessing it shouldn't.

Reply:
It shouldn't, the reason is because the script copies the entire contents of that directory when the docker image is built, so if you attempted to build the application and then create the docker image, the build artifacts will be copied.

I'll modify the installation script to run a clean before the build to make sure any build artifacts are cleaned up.

Reply:
Oh, right. Good catch.